webviews/components/main_panel.svelte

```diff
@@ -0,0 +1,67 @@
+<script>
+  const data = {}
+  const ids = (e) => {
+    const formData = new FormData(e.target)
+
+    for (let field of formData) {
+      const [key, value] = field
+      data[key] = value
+    }
+    console.log(data)
+  }
+
+  const historyFile = (e) => {
+    const formData = new FormData(e.target)
+
+    for (let field of formData) {
+      const [key, value] = field
+      data[key] = value
+    }
+    console.log(data)
+  }
+
+  const downloadHistory = (e) => {
+    const formData = new FormData(e.target)
+
+    for (let field of formData) {
+      const [key, value] = field
+      data[key] = value
+    }
+
+    vscode.postMessage({
+      type: "onSubmit",
+    })
+    console.log(data)
```

Comment:
I'd add some comments here explaining that this is work in progress and what it should eventually do.